scripts/screener/screener.types.ts

```diff
@@ -2,22 +2,22 @@ export type ScreenerRunnerConfig = {
   apiKey: string;
   projectRepo: string;
 
-  diffOptions: {
+  diffOptions?: {
     structure: boolean;
     layout: boolean;
     style: boolean;
     content: boolean;
     minLayoutPosition: number; // Optional threshold for Layout changes. Defaults to 4 pixels.
     minLayoutDimension: number; // Optional threshold for Layout changes. Defaults to 10 pixels.
     minShiftGraphic: number; // Optional threshold for pixel shifts in graphics.
-    compareSVGDOM: number; // Pass if SVG DOM is the same. Defaults to false.
+    compareSVGDOM: boolean; // Pass if SVG DOM is the same. Defaults to false.
```

Comment:
This was always typed incorrectly before applying the types to the screener.config.js